File: 11247/income.py
import sys

#Accepted

for line in sys.stdin:
    t = line.strip().split()

    m = int(t[0])
    x = int(t[1])

    if m == 0 and x == 0:
        break

    if x == 0 or x == 100:
      print("Not found")
      continue

    v = ((m - 1)*100) // (100 - x)
    temp = ((m - 1)*100) % (100 - x)

    if temp == 0:
        v = v - 1

    if v < 0 or v < m:
        print("Not found")
        continue

    print(v)

File: 11247/income.c
#include <stdio.h>

//Accepted
int main(int argc, char *argv[])
{
  long long m, x, v;
  long long t;
  double temp;

  while(1)
  {
    scanf("%lld %lld", &m, &x);

    if(m == 0 && x == 0)
      break;

    if(x == 0 || x == 100)
    {
      printf("Not found\n");
      continue;
    }

    v = ((m - 1) * 100) / (100 - x);
    t = ((m - 1) * 100) % (100 - x);

    if (t == 0)
      v--;

    if(v < 0 || v < m)
    {
      printf("Not found\n");
      continue;
    }
    printf("%lld\n", v);
  }

  return 0;
}
